index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Mobile Skeleton</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+        <link rel = "stylesheet" href = "styles/reset.css" type = "text/css" media = "all">
+        <link rel = "stylesheet" href = "styles/style.css" type = "text/css" media = "all">        
+    </head>
+    <body>
+      <div class="device-container">
+        <div class="device-look">
+          <header>
+            <a class="logo-container" href="#">
+                <span class="logo">Welcome</span>
+            </a>           
+            <nav class="menu-button">
+                <a class="menu-anchor" href="#">
+                    <img src="images/menu.png" width="20" height="20" alt="Menu" title="Menu">
+                </a>                
+              </nav>
+          </header>
```

Comment:
- [ ] Kindly add section margin-top to make your design follow the Figma template as the header section is currently not visible
Your design
![jesus](https://user-images.githubusercontent.com/85095702/158901934-1e36930c-610f-4c99-a401-80a371be64b4.PNG)

Figma template
![Capture](https://user-images.githubusercontent.com/85095702/158902168-5247922b-7675-4419-af4f-5b1aed0e13d2.PNG)

Reply:
First of all, thank you for the prompt response. I only have one issue, the header actually appears in my browser, and I already tested it on another computer and the header is still showing as well. Maybe you have to zoom out your browser so you can see the whole page.

My Laptop (ubuntu):
![image](https://user-images.githubusercontent.com/60033459/158912759-6db1463e-bafd-4070-8648-322ea7a9ab56.png)

Another laptop (windows):
![screenshot-mobile-skeleton](https://user-images.githubusercontent.com/60033459/158913030-d7b3bf34-e85c-4f0e-8c95-785a685bba1d.jpeg)

I suspect about the zoom-out thing because you're screenshot isn't showing the up-scroll button:

![reviewer_screenshot](https://user-images.githubusercontent.com/60033459/158914439-5f94f5e4-3c9b-4b53-b603-962de92d0d72.jpeg)

Or perhaps you could share to me a full screenshot to see what seems to be the problem.

I tried to add a "margin-top" property but it unsettles the remaining elements, look:
![image](https://user-images.githubusercontent.com/60033459/158915182-965fdc4e-d1ae-42af-a22e-677c8ac6641e.png)

And this is the original configuration:

![image](https://user-images.githubusercontent.com/60033459/158915457-34cf95e6-a798-438b-9b13-f53f3ab7eb81.png)

Thanks in advance, I look forward to your comments. 🤓👍

Reply:
You did a great job with the project, I've tested it on another browser, and it's still not visible at 100% zoom, but became visible at say 90% zoom...but you don't want to be asking all your users to zoom out every time they want to use your webpage 🤓...try adding the margin-top to the `<header>` tag instead of the `.headline` class... I've tried it and it works fine.

Reply:
And also, when adding `width, height, and margins` always favor units that are percent base (vh, vw, %) instead of (px) as they make your webpage more responsive. 👍